src/AutoRest.CSharp/Common/Input/CadlSerialization.cs

```diff
@@ -30,6 +30,9 @@ internal static class CadlSerialization
             options.Converters.Add(new CadlInputEnumTypeValueConverter(referenceHandler));
             options.Converters.Add(new CadlInputModelTypeConverter(referenceHandler));
             options.Converters.Add(new CadlInputModelPropertyConverter(referenceHandler));
+            options.Converters.Add(new CadlInputConstantConverter(referenceHandler));
+            options.Converters.Add(new CadlInputAuthConverter(referenceHandler));
```

Comment:
Do we need custom converter for `InputAuth`? I'd expect such input type to be handled automatically.

Reply:
Yes, we need custom converter. We emit `InputAuth` as an object with id, and other two fields(ApiKey, OAuth2). Need customer converter to handle it.

Reply:
Ok, I'll check it myself